packages/adaptive_scaffold/example/README.md

```diff
@@ -0,0 +1,9 @@
+# Examples
```

Comment:
Not sure if this is intentional, but this is missing the Windows entry point. If that's not intentional, you can add this by running ` flutter create . --platforms=windows`

Reply:
Thanks just added this.